core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpInsertOtherTypesTest.java

```diff
@@ -865,11 +865,13 @@ public void testInsertTimestamp() throws Exception {
     public void testInsertTimestampNS() throws Exception {
         timestampType = TestTimestampType.NANO;
         assertTimestamp(
-                "value\ttimestamp\n" +
```

Comment:
test calls parameterized method with the same value, it is either redundant test or redundant parameter, please investigate, there is warning in this file

Reply:
It is parametrized with WAL/non-WAL, not with micro/nano.
`testInsertTimestampNS()` runs the same thing as `testInsertTimestamp()`, but with `TestTimestampType.NANO`.
This test asserts that V1 protocol always uses micros precision, even with `TestTimestampType.NANO`.